components/CreatorList.tsx

```diff
@@ -46,13 +44,23 @@ const CreatorList: React.FC<Props> = ({ title, params, enabled, narrow = false,
 		<div className='creator-list-wrapper'>
 			<h2 className='title'>{title}</h2>
 			{hasCreators && (
-				<Grid container spacing={1} className='creator-list'>
-					{creators.map((creator) => (
-						<Grid item key={creator.slug} xs={6} md={4} lg={3} xl={2}>
-							<CreatorItem creator={creator} key={creator.slug} />
-						</Grid>
-					))}
-				</Grid>
+				<table>
+					<thead>
+						<tr>
+							<td>Name</td>
+							<td>Email</td>
+							<td>Avatar</td>
+							<td>Logo</td>
+							<td>Banner</td>
+							<td>Twitter</td>
+							<td>Is Email Verified</td>
+							<td>Is Verified</td>
+						</tr>
+					</thead>
+					<tbody>
+						{creators.map((creator)=> <RawCreatorItem key={creator.name} creator={creator}/>)}
```

Comment:
Leave this be as it is for now, but you should never ever put <td> or <tr> within some component as the base element.

If you do end up doing it... call the component something like `RawCreatorTableRow` instead, as the name better reflects what's within the component.

If I see `RawCreatorItem` I expect a `div` or in the worst case `li` element. I personally would name the component `RawCreatorListItem` if I expected a `li`, and `RawCreatorItem` if it's just some basic `div` item.

Even Mui calls `li` components as `ListItem` (https://mui.com/material-ui/api/list-item)